index.js

```diff
@@ -942,7 +977,7 @@
   //. > S.init([])
   //. Nothing()
   //. ```
-  S.init = R.ifElse(R.isEmpty, Nothing, R.compose(Just, R.init));
+  S.init = S.slice(0, -1);
```

Comment:
That's really elegant!

I've not really looked, but I haven't seen many truly useful reasons to have `-0` except that the number implementation demanded it.  This shows that there really can be such.  Nice!